contrib/examples/actions/workflows/mistral-yaql-st2kv-user-scope.yaml

```diff
@@ -0,0 +1,12 @@
+version: '2.0'
+
+examples.mistral-yaql-st2kv-user-scope:
+    vars:
+        polo: unspecified
+    tasks:
+        task1:
+            action: std.noop
+            publish:
+                polo: <% st2kv('marco') %>
```

Comment:
YAQL evaluate any dot as a dict. So st2kv.get() won't work. st2kv[key] will assume st2kv is a list because of the bracket. Otherwise we will have to hack YAQL to achieve this result. Also, YAQL is read/eval only. I don't recommend doing set in YAQL even if it's possible. That will be very inconsistent as anyplace where YAQL is evaluated in mistral will now be able to write out to st2 datastore. I prefer not to support this. User should explicitly have an action to set the key. Given that YAQL is read/eval only, it's not necessary to explicitly say st2kv_get or st2kv[key]. I think st2kv() as a function is fine the way it is.